docs/next/guides/getting-started/installation.md

```diff
@@ -11,20 +11,65 @@ tags:
 
 [[toc]]
 
-## Overview
-This guide details how to install Handsontable.
+Install Handsontable in your preferred way.
 
 ::: tip
-This section is dedicated to the pure JavaScript version of Handsontable. If you use a framework in your project, follow one of the available guides to install and use the library:
- - [Integrate with React](@/guides/integrate-with-react/react-installation.md)
- - [Integrate with Angular](@/guides/integrate-with-angular/angular-installation.md)
- - [Integrate with Vue 2](@/guides/integrate-with-vue/vue-installation.md)
- - [Integrate with Vue 3](@/guides/integrate-with-vue3/vue3-installation.md)
+To use Handsontable with a framework, see:
+
+ - [Installation in React](@/guides/integrate-with-react/react-installation.md)
+ - [Installation in Angular](@/guides/integrate-with-angular/angular-installation.md)
+ - [Installation in Vue 2](@/guides/integrate-with-vue/vue-installation.md)
+ - [Installation in Vue 3](@/guides/integrate-with-vue3/vue3-installation.md)
 :::
 
-## Download and install the library
+## Overview
+
+<code-group>
+  <code-block title="npm">
+
+  ```bash
+  npm install handsontable
+  ```
+
+  </code-block>
+  <code-block title="Your HTML">
+
+  ```html
+  <div id="your-container"></div>
+  ```
+
+  </code-block>
+  <code-block title="Your application">
+
+  ```js
+  import Handsontable from 'handsontable';
+  
+  const container = document.getElementById('your-container');
+  
+  const yourHandsontableInstance = new Handsontable(container, {
+    data: Handsontable.helper.createSpreadsheetData(5, 5),
+    licenseKey: 'non-commercial-and-evaluation' // for non-commercial use only
+  });
```

Comment:
I have a few comments to this snippet:

1. It is confusing that this snippet is different than the other snippet down the page. 
2. It is missing the importing of the stylesheet.
3. The instance name `hot` is a convention used on 88 pages. I would love to change it to `handsontable`, but only on all pages at once. The name `handsontable` would be good, because it is an established convention that an instance of a class has the same name as the class but in lowercase.
4. We should avoid showing off `createSpreadsheetData`, for the following reasons:
   - it's function is unknown to the reader of this page
   - the "data grid" scenario should be the promoted one. Soon most of the snippets will be converted to "an array of objects"
   - `createSpreadsheetData` is a function that we mostly use internally for the tests and quick demos, but it's teaching value is doubtful

----

I suggest to just paste the snippet found at the bottom of the page:

```suggestion
import Handsontable from "handsontable";
import "handsontable/dist/handsontable.full.css";

const data = [
  ["", "Tesla", "Volvo", "Toyota", "Ford"],
  ["2019", 10, 11, 12, 13],
  ["2020", 20, 11, 14, 13],
  ["2021", 30, 15, 12, 13],
];

const container = document.getElementById("example");

const hot = new Handsontable(container, {
  data: data,
  rowHeaders: true,
  colHeaders: true,
  height: "auto",
  licenseKey: "non-commercial-and-evaluation", // for non-commercial use only
});
```